agents-api/agents_api/routers/sessions/chat.py

```diff
@@ -144,16 +250,28 @@ async def chat(
         for m in messages
     ]
 
-    # Get the response from the model
-    model_response = await litellm.acompletion(
-        messages=messages,
-        tools=tools or None,
-        user=str(developer.id),  # For tracking usage
-        tags=developer.tags,  # For filtering models in litellm
-        custom_api_key=x_custom_api_key,
-        **settings,
+    # Check if using Claude model and has specific tool types
+    is_claude_model = settings["model"].lower().startswith("claude-3.5")
+    has_special_tools = any(
+        tool["type"] in ["computer_20241022", "bash_20241022", "text_editor_20241022"]
+        for tool in formatted_tools
     )
 
+    if is_claude_model and has_special_tools:
+        model_response = await request_anthropic(messages, formatted_tools, settings)
+    else:
+        # FIXME: hardcoded tool to a None value as the tool calls are not implemented yet
```

Comment:
Is all of this safe to merge? @Ahmad-mtos @HamadaSalhab

Reply:
no we removed the anthropic client from this file